docs/basic_construction.md

```diff
@@ -0,0 +1,125 @@
+# Telescope - Basic Construction
+**ALBA** provides a method for a prover to convince a verifier that the prover knows at more than $n_f$ elements satisfying a predicate $R$, while only presenting a small subset of elements as proof. 
+This construction uses a recursive filtering process, guided by random oracles, to produce succinct and efficient proofs.
+
+## Overview
```

Comment:
Perhaps the first and last sentences could be outside the itemize and the rest in a numbered list?
It feels like you are cramming together different things: how the protocol works (step by step) and information about the oracle and so on.